eclipse-collections/src/main/java/org/eclipse/collections/impl/list/Interval.java

```diff
@@ -1063,7 +1063,7 @@ public Interval take(int count)
         {
             return Interval.fromToBy(this.from, this.locationAfterN(count - 1), this.step);
         }
-        return Interval.fromToBy(this.from, this.from, this.step);
```

Comment:
```
Interval interval = Interval.fromTo(1, 4);
Assert.assertEquals(Lists.immutable.empty(), interval.take(0));

```
Result:
`java.lang.AssertionError: `
Expected :[]
Actual   :Interval from: 1 to: 1 step: 1 size: 1